src/EditorServicesCommandSuite/CodeGeneration/Refactors/CommandSplatRefactorSettings.cs

```diff
@@ -12,5 +12,17 @@ internal class CommandSplatRefactorSettings : RefactorConfiguration
         [Parameter]
         [DefaultFromSettingAttribute("CommandSplatRefactor.NewLineAfterHashtable", Default = "$true")]
         public SwitchParameter NewLineAfterHashtable { get; set; }
+
+        [Parameter]
+        [DefaultFromSettingAttribute("CommandSplatRefactor.AllParameters", Default = "$false")]
```

Comment:
`AllParameters` is kind of ambiguous about what it will do, but I'm stuck on what this should be named.  I'm leaning towards `ResolveAdditionalParameters`.  Any thoughts?